lib/avtp_pipeline/platform/Linux/rawsock/pcap_rawsock.c

```diff
@@ -66,9 +64,9 @@ void *pcapRawsockOpen(pcap_rawsock_t* rawsock, const char *ifname, bool rx_mode,
 	// Deal with frame size.
 	if (rawsock->base.frameSize == 0) {
 		// use interface MTU as max frames size, if none specified
-		rawsock->base.frameSize = rawsock->base.ifInfo.mtu;
+		rawsock->base.frameSize = rawsock->base.ifInfo.mtu + ETH_HLEN + VLAN_HLEN;
 	}
-	else if (rawsock->base.frameSize > rawsock->base.ifInfo.mtu) {
+	else if (rawsock->base.frameSize > rawsock->base.ifInfo.mtu + ETH_HLEN + VLAN_HLEN) {
```

Comment:
It does make sense to me to include the size of the vlan+ethernet headers here. Did we have a bug before on these lines?